docs/conf.py

```diff
@@ -164,12 +164,19 @@ def autodoc_skip_member(app, what, name, obj, skip, options):
     return name in exclude_pyapi_methods
 
 
+shutil.copy("../../../docs/home.rst",".")
+
+def replace_index_with_redirect(app,exception):
+    print(os.getcwd())
```

Comment:
What is the intention of this print? Is it helpful in other way than just debug this function?

Reply:
Thanks for catching these. Should be fixed now.